### DISCUSSION TITLE
2023-12-28

### DISCUSSION THREAD

**Discussion by yutingzhao1991** [ant-design/ant-design-web3#411] [中文社区双周会]
请回复待讨论的话题并记录相关纪要。

**Comment by yutingzhao1991** [comment]
1.0 发布后要做的关键事项：https://github.com/ant-design/ant-design-web3/discussions/370

**Reply by yutingzhao1991** [reply]
hot chain:

**solana**
near
sui
cosmos

热点看情况跟进：
- 比特币铭文
- twiter 电子宠物

**Comment by yutingzhao1991** [comment]
关于支持更多钱包通过 WalletConnect 连接的方案设计：https://github.com/ant-design/ant-design-web3/issues/414

**Comment by jeasonstudio** [comment]
连接 Solana Chain 可以参考 Cookbook：[solana cookbook](https://solanacookbook.com/references/keypairs-and-wallets.html#how-to-connect-to-a-wallet)
看起来用法很接近 wagmi，它也提供了一些高阶用法，比如 react hooks

链 Icon 可以参考：https://cryptologos.cc/

**Comment by yutingzhao1991** [comment]
会议纪要：

设计相关时间节点：
- 设计稿规范，走查，优化 1 月。
- 钱包连接弹窗优化，新的链接模式，2 月设计，3 月中完成。
- 穿插 icon 补充，1 - 4 月补充完成主流的链和代币。
- 新组件，3 月设计，4 月支持。S1 2-3 个组件。

---

待办任务：

1. 支持 solana @gin-lsl @li-jia-nan @jeasonstudio  下次双周会完成基本功能架子搭建。
2. 弹窗优化 @yutingzhao1991 下次双周会产出需求，钱包具体逻辑分享。
3. icon @Likang0122 整理需求，下次双周会前完成。
4. 教程 @jeasonstudio 第一部分，下次双周会前完成。
5. 国际化 @yutingzhao1991 找 @kiner-tang 确认下 ConnectModal 进展。#416
6. https://github.com/ant-design/ant-design-web3/issues/414 @yutingzhao1991 下周跟进。
